site/pages/rest-api/examples/update-site.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Update Site
+---
+
+Updates an existing site on the server with the contents of `site.xml`.
+
+You can get the list of sites with the following commands:
+
+# cURL
```

Comment:
If anything I think this can be moved to a separate md file called "Get Sites". Since this is a reference page, conciseness is important here, so all the examples should be relevant to "update sites"